action.yml

```diff
@@ -19,7 +19,7 @@ inputs:
     required: false
     default: balena-cloud.com
   versionbot:
-    description: Does the repository use VersionBot ?
+		description: (Deprecated) Does the repository use VersionBot?
```

Comment:
spacing

Reply:
Some weird code editor behaviour going on, keeps reverting back to tabs instead of spaces